tests/integration/flows/test_mysql_api_extended.py

```diff
@@ -0,0 +1,402 @@
+import os
+import pytest
+import uuid
+import time
+from .test_mysql_api import BaseStuff
+import mysql.connector
+
+
+@pytest.fixture(scope="module")
+def setup_local_db():
+    """Module-scoped fixture to create a writeable DB for table tests."""
+    db_name = f"test_db_{uuid.uuid4().hex[:8]}"
+    helper = BaseStuff()
+    helper.use_binary = False
+
+    params = {"user": "postgres", "password": "postgres", "host": "postgres", "port": 5432, "database": "postgres"}
```

Comment:
just noticed that there is "postgres" host. will it be available in CI environment?